docs/guides/packer-bucket-rbac.md

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: ""
+page_title: "Managing Packer Bucket IAM Policies"
+description: |-
+    A guide to using HCP Packer bucket resource along with binding or policy resource to manage bucket level access.
+---
+
+# Managing Packer Bucket IAM Policies
+
+You can grant specific users, service principals, or groups contributor or admin level access to a specific HCP Packer bucket using either a `hcp_packer_bucket_iam_binding` or `hcp_packer_bucket_iam_policy` resource.  Whenever a user is invited to a project they will have read level access to all resources, but you can restrict which of the principals in your project can maintain specific buckets.
+
+A resource's policy is a list of bindings to assign roles to multiple users, groups, or service principals. The `hcp_packer_bucket_iam_policy` resource sets the Bucket IAM policy and replaces any existing policy.
+
+The following example assigns the role `contributor` to a user principal and a service principal for the `production` bucket.
+
+```terraform
+data "hcp_iam_policy" "mypolicy" {
+  bindings = [
+    {
+      role = "roles/contributor"
+      principals = [
+        "user-principal-id-1",
+        "service-principal-id-1",
+      ]
+    },
+  ]
+}
+
+resource "hcp_packer_bucket" "production" {
+  name = "production"
+}
+
+resource "hcp_packer_bucket_iam_policy" "example" {
+  resource_name = hcp_packer_bucket.production.resource_name
+  policy_data   = data.hcp_iam_policy.mypolicy.policy_data
+}
+```
+
+A resource's policy binding assigns a single role to a list of principals. The `hcp_packer_bucket_iam_binding` resource updates the Bucket IAM policy to bind a role to a new member. Existing bindings are preserved.
```

Comment:
When reading the first phrase, it sounds like the next resource will assign a role to a list of principals, similar to the one before. So to avoid confusion I would jump to the next sentence that contains enough information. What do you think? 
```suggestion
The `hcp_packer_bucket_iam_binding` resource updates the Bucket IAM policy to bind a role to a new member. Existing bindings are preserved.
```

Reply:
All of your suggested changes make sense to me, I have committed them in a separate commit, this file `packer-bucket-rbac.md` is generated from a file with a .tmpl extension in the templates directory, resolving these suggestions